### DISCUSSION TITLE
Love that Feature

### DISCUSSION THREAD

**Discussion by ptal78** [BIPES/BIPES#51] [General]
Bipes is a great tool to write code quick and easy.
We know great things take time, but the team here is super helpful and they work hard to get the job done.

My favourite feature is the **NeoPixel blocks for the ESP32**. By adding variables to change the hue, our kids have great fun coding.
There are so many features, and I'm keen to try them all out with my school.

**What's your favourite feature?**

**Comment by rafaelaroca** [comment]
About my favorite feature, I love the quick and easy way to connect several boards using USB or Wifi and quickly program, monitor, debug and experiment without any special software or compiler installed on my computer.

**Reply by jibeer** [reply]
Totally agree : BIPES is super fast to programm IOT, without installing a mess on our computers....

**Comment by rafaelaroca** [comment]
The new feature of splitting the screen is one that I love!

https://github.com/BIPES/BIPES/discussions/76
